ci/vmtest/run_selftests.sh

```diff
@@ -27,6 +28,21 @@ read_lists() {
 	done) | cut -d'#' -f1 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' | tr -s '\n' ','
 }
```

Comment:
WE should get rid of this now that https://lore.kernel.org/bpf/20230427225333.3506052-1-sveiss@meta.com/ is implemented.

I am not sure it would take multiple files, but `-a @<(cat  "$BPF_SELFTESTS_DIR/ALLOWLIST" "$BPF_SELFTESTS_DIR/ALLOWLIST.${ARCH}" "$VMTEST_CONFIGS_PATH/ALLOWLIST" "$VMTEST_CONFIGS_PATH/ALLOWLIST.${ARCH}")` should work